buffalo/cmd/dev.go

```diff
@@ -2,11 +2,11 @@ package cmd
 
 import (
 	"context"
-	"fmt"
 	"os"
 	"os/exec"
 	"runtime"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?